### PR TITLE
Fix import of expiration date from LastPass

### PR DESCRIPTION
**Pull request by felipecassiors**
resolves https://github.com/bitwarden/web/issues/443

### CODE REVIEW

**Comment by felipecassiors** [comment]
This still needs testing that I don't know how to do.

File: src/importers/lastpassCsvImporter.ts [hunk 1]
```diff
@@ -166,11 +166,23 @@ export class LastPassCsvImporter extends BaseImporter implements Importer {
...
-                    const mappedData = this.parseSecureNoteMapping<CardView>(extraParts, {
+                    let mappedData = this.parseSecureNoteMapping<CardView>(extraParts, {
                         'Number': 'number',
                         'Name on Card': 'cardholderName',
                         'Security Code': 'code',
+                        'Expiration Date': 'expMonth',
                     });
+
+                    // Split string 'January,2020' into two
+                    const tempExpDate = mappedData[0].expMonth.split(',');
```
Comment (kspearrin) [inline src/importers/lastpassCsvImporter.ts]: Can `mappedData[0].expMonth` be null/undefined? Seems you need some null checking here, right?

Comment (felipecassiors) [reply src/importers/lastpassCsvImporter.ts]: That's right. Done.

**Comment by kspearrin** [comment]
https://github.com/bitwarden/jslib/pull/65 was merged, which builds from this.

**Pull request closed without merging**